### ISSUE TITLE
acc does't improve while training 100 epochs

### ISSUE THREAD

**Issue by eternal8080** [yelboudouri/EmoNeXt#4]
![image](https://github.com/yelboudouri/EmoNeXt/assets/76619240/2bf1c673-b57e-4ce8-953e-c3c520a2afc7)
training loss, valid acc don' improve

**Comment by dastansultanov** [comment]
@eternal8080
Hello, have you found a way to solve the problem, because I am facing the same problem
@yelboudouri could you please give us the right direction to solve the problem?

**Comment by eternal8080** [comment]
I haven't solve this problem

**Comment by L-yi-yi** [comment]
我的训练结果也是这样

**Comment by eternal8080** [comment]
problem solved， set learn rate to 1e-5， then it will be ok

**Comment by dastansultanov** [comment]
@eternal8080 could you please share configs you've used? batch-size, lr and etc.

**Comment by eternal8080** [comment]
lr=1e-5，and in scheduler.py set max_lr: float = 1e-5, other configs remain unchanged

**Comment by dastansultanov** [comment]
@eternal8080 thanks, it helped to solve the issue. What was your accuracy metric score? Did you achieve an accuracy of 76.12%?

**Comment by eternal8080** [comment]
my best accuracy was 73.6% on base version

**Comment by f15536** [comment]
> lr=1e-5，and in scheduler.py set max_lr: float = 1e-5, other configs remain unchanged

Hello, did you only change lr in scheduler.py? or lr in train.py needs to be changed, too?  thx
